src/DynamoPackages/PackageDocManager.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.PackageManager
+{
+    /// <summary>
+    /// Creates a lookup for node annotation markdown files associated with a package.
+    /// Also creates file watchers for the package node annotation paths so new files can be hot reloaded in Dynamo.
+    /// </summary>
```

Comment:
Could these capabilities be moved to the `DocumentationBrowserViewExtension`? This is all triggered from the Package Manager's OnPackageLoaded handler; the doc browser extension could subscribe to that event as well. Also, this looks like we're monitoring all documentation for all packages at all times -- that seems excessive. We want to watch the minimum necessary to make hot reload work. It's okay to defer a lot of this work until the user requests help for a node.

Reply:
@Dewb we are watching all packages `doc` folder so we know if the documentation is added or some is removed, but we are only watching the current open documentation file for changes. I cant see the way to defer that to when the user press help, or am i missing something?